### PR TITLE
Bump Dotty to RC1

### PR DESCRIPTION
**Pull request by anatoliykmetyuk**

### CODE REVIEW

**Review by alexarchambault — COMMENTED** [review]
Thanks! There are other files to update for Scala 3, namely all the `scala3-*` ones in the same directory.

**Comment by alexarchambault** [comment]
As an aside, I really wish I could automate that version handling in coursier, so that the line you changed would become
```
"org.scala-lang:::scala3-compiler:latest.release"
```

File: apps/resources/scala3-doc.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "central"
   ],
   "dependencies": [
-    "org.scala-lang:scala3-doc_3.0.0-M3:3.0.0-M3"
+    "org.scala-lang:scala3-doc_3.0.0-RC1:3.0.0-RC1"
```
Comment (anatoliykmetyuk) [inline apps/resources/scala3-doc.json:7]: @romanowski is this correct? What should the main class name be instead of `dotty.tools.dottydoc.Main`?

Comment (romanowski) [reply apps/resources/scala3-doc.json:7]: in RC1 we've moved to calling it just `scaladoc`. It should use `dotty.tools.scaladoc.Main` for cmd line version for sbt-like it is `dotty.tools.dottydoc.Main`

**Comment by anatoliykmetyuk** [comment]
@alexarchambault should be good now.

**Review by alexarchambault — APPROVED** [review]
Sorry for the delay… Thanks @anatoliykmetyuk!

**Pull request merged**